### ISSUE TITLE
EDL files not getting created properly

### ISSUE THREAD

**Issue by dailow** [erikkaashoek/Comskip#110]
Most of the time EDL files are created properly, but sometimes there is some extra output being produced that is breaking the functionality when Kodi tries to process it.  Any idea where the extra text is coming from?

`After - 3.00
Block 33 has no logo.
Block 33 score:	Before - 3.00	After - 6.00
CC's wrong type - Block 33 score:	Before - 6.00	After - 12.00
Block 34 has non-strict standard length for a commercial.
Block 34 score:	Before - 1.00	After - 1.50
Block 34 has no logo.
Block 34 score:	Before - 1.50	After - 3.00
CC's wrong type - Block 34 score:	Before - 3.00	After - 6.00
Block 35 has strict standard length for a commercial.
Block 35 score:	Before - 1.00	After - 3.00
Block 35 has no logo.
Block 35 score:	Before - 3.00	After - 6.00
No CC's - Block 35 score:	Before - 6.00	After - 9.00
Block 36 has no logo.
Block 36 score:	Before - 1.00	After - 2.00
CC's correct type - Block 36 score:	Before - 2.00	After - 1.50
Block 37 has logo.
Block 37 score:	Before - 1.00	After - 0.01
CC's correct type - Block 37 score:	Before - 0.01	After - 0.01
Dictionary not processed successfully
Threshold used - 1.0500	After rounding - 1.0500

	---------------------
	Initial Commercial List
	---------------------
 0)      1	   878	0:00:29.26
 0.00	29.26	3
53.62	211.14	3
1030.90	1151.04	3
1459.95	1651.51	3
2029.35	2194.49	3
2394.98	2610.23	3
2964.61	3028.94	3
3268.63	3652.37	3
`

**Comment by erikkaashoek** [comment]
What you show is a part of the log file preceding the content of the EDL file. Is all coming from an EDL file?

**Comment by dailow** [comment]
It is all coming into the EDL file.  I noticed all this is coming from Debug statements, so I set verbose=0 to see if this helps.  I also don't have any other debug flags set.

**Comment by erikkaashoek** [comment]
Seems the output log and the edl output are written to the same file.
When you set verbose above zero, is there a log file created?

**Comment by dailow** [comment]
It does not create a log file with verbose set to 0.  Will have to run it for a few days to see if it solves the problem.

Not a C expert, but looking through some of the code it seems like Debug function has its own file for logging, not sure how it would get mixed into the edl_file

**Comment by erikkaashoek** [comment]
Indeed, tt should only create a separate log file with verbose > 0 and this should never end up in the EDL file unless we have a memory corruption
